CinemaTicketSalesSystem/CinemaTicketSalesBusinessLogic/Queries/DbService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ApplicationDbMovies.Contexts;
+using ApplicationDbMovies.Models;
+
+namespace CinemaTicketSalesBusinessLogic.Queries
+{
+    public static class DbService
+    {
+        public static ApplicationDbContext db = new ApplicationDbContext();
+
+        /// <summary>
+        /// Using in HomeController for viewing Movies collection on Main page.
+        /// </summary>        
+        public static IEnumerable<MovieViewModel> GetMovieInfo()
```

Comment:
MovieViewModel shouldn't be returned here.
In 3-layer architecture you should have models for each leayer.

For example:
DAL - Movie
BL - MovieDto
Web - MovieViewModel

And you should return BL model from BL layer method.